types/react-beautiful-dnd/v10/index.d.ts

```diff
@@ -192,6 +192,11 @@ export interface DraggableStateSnapshot {
     mode?: MovementMode;
 }
 
+export interface Position {
```

Comment:
Where is this used and why does it need to be added?

Reply:
It used below:
```
export interface Position {
    x: number;
    y: number;
}

export interface DropAnimation {
    duration: number;
    curve: string;
    moveTo: Position;
    opacity?: number;
    scale?: number;
}
```
There was used wrong type Position, there is example of usage:
https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/drop-animation.md

Reply:
omfg